src/templates/admin/journal/manage/add_guest_editor.html

```diff
@@ -15,111 +15,123 @@
+                        <th>Volume</th>
+                        <th>Number</th>
+                        <th>Date</th>
+                        <th>No. Articles</th>
+                    </tr>
+                    <tr>
+                        <td>{{ issue.issue_title }}</td>
+                        <td>{{ issue.volume }}</td>
+                        <td>{{ issue.issue }}</td>
+                        <td>{{ issue.date|date:"Y-m-d" }}</td>
+                        <td>{{ issue.manage_issue_list|length }}</td>
+                    </tr>
+                </table>
             </div>
         </div>
+    </div>
 
-        <div class="large-12 columns">
-            <div class="box">
-                <div class="title-area">
-                    <h2>Guest Editors</h2>
-                </div>
-                <div class="content">
+    <div class="large-12 columns">
+        <div class="box">
+            <div class="title-area">
+                <h2>Guest Editors</h2>
+            </div>
+            <div class="content">
+                <form method="POST">
+                    {% csrf_token %}
                     <table class="small scroll">
                         <thead>
                         <tr>
                             <th>ID</th>
                             <th>First Name</th>
                             <th>Last Name</th>
                             <th>Email</th>
+                            <th>Role</th>
                             <th></th>
                         </tr>
                         </thead>
                         <tbody id="sortable">
-                        {% for user in guest_editors %}
-                            <tr id="articles-{{ dict.article.pk }}">
-                                <td>{{ user.id }}</td>
-                                <td>{{ user.first_name }}</td>
-                                <td>{{ user.last_name }}</td>
-                                <td>{{ user.email }}</td>
+                        {% for editor in editors %}
+                            <tr>
+                                <td>{{ editor.account.id }}</td>
+                                <td>{{ editor.account.first_name }}</td>
+                                <td>{{ editor.account.last_name }}</td>
+                                <td>{{ editor.account.email }}</td>
                                 <td>
-                                    <button type="submit" name="user_remove" value="{{ user.pk }}"><i class="fa fa-chain-broken">
-                                        &nbsp;</i>Remove
+                                    {{ editor.role }}
+                                </td>
+                                <td>
+                                    <button type="submit"
+                                            name="user_remove"
+                                            value="{{ editor.pk }}"
+                                            formaction="{% url 'manage_remove_issue_editor' issue.pk %}"
```

Comment:
I found this is a bit confusing, why do we use two different forms for adding/removing if we are overriding the action on the submit anyways? I think it would be clearer to either the action to the <form> element

Reply:
Separate forms are to avoid posting the contents of all of the text input when deleting. I've moved the formaction to the <form> action tag.